src/inmanta_ui/config.py

```diff
@@ -17,19 +17,30 @@
 """
 from inmanta.config import Option, is_bool, is_str
 
-web_console_enabled = Option("web-console", "enabled", True, "Should the server should host the web-console or not", is_bool)
+web_console_enabled = Option(
+    "web-ui", "console_enabled", True, "Whether the server should host the web-console or not", is_bool
+)
 web_console_path = Option(
-    "web-console",
-    "path",
+    "web-ui",
+    "console_path",
     "/usr/share/inmanta/web-console",
     "The path on the local file system where the web-console can be found",
     is_str,
 )
 web_console_json_parser = Option(
-    "web-console",
-    "json_parser",
+    "web-ui",
+    "console_json_parser",
     "Native",
     "Whether the web-console should use the 'Native' or the 'BigInt' JSON Parser. "
     "'BigInt' is useful when the web-console has to show very large integers (larger than 2^53 - 1).",
     is_str,
 )
+#############################
+# Keycloak authentication
+#############################
+
+keycloak_realm = Option("web-ui", "keycloak_realm", "inmanta", "The realm to use for keycloak authentication.", is_str)
+keycloak_auth_url = Option("web-ui", "keycloak_auth_url", None, "The auth url of the keycloak server to use.", is_str)
+keycloak_client_id = Option(
+    "web-ui", "keycloak_client_id", None, "The client id configured in keycloak for this application.", is_str
+)
```

Comment:
@bartv Do we need to prefix these options to make clear that they belong to the web_console?

Reply:
We have never tested it but it should work with generic open id connect. So lets prefix them with with `oidc_`